bindings/python/python_constants.cpp

```diff
@@ -389,7 +389,11 @@ void addLabels(PyObject * dict)
     ADD_CONST(MDL_SCORE_BY_VAR);
     ADD_CONST(MDL_SCORE_BY_ZERNIKE);
     ADD_CONST(MDL_SCORE_BY_ZSCORE);       
-              
+    ADD_CONST(MDL_SPH_COEFFICIENTS);
+    ADD_CONST(MDL_SPH_DEFORMATION);
+    ADD_CONST(MDL_SPH_TSNE_COEFF1D);
+    ADD_CONST(MDL_SPH_TSNE_COEFF2D);
```

Comment:
This four constants must be defined in the xmippCore repository. Please, make a pull request in the xmippCore repository where they are included.